tests/fastcs/panda/test_hdf_panda.py

```diff
@@ -134,134 +134,8 @@ def flying_plan():
             )
         yield from bps.wait(group="complete")
         yield from bps.close_run()
-
-        yield from bps.unstage_all(flyer, mock_hdf_panda)
-        yield from bps.wait_for([lambda: mock_hdf_panda.controller.disarm()])
-
-    # fly scan
-    RE(flying_plan())
-
-    assert_emitted(
-        docs, start=1, descriptor=1, stream_resource=2, stream_datum=2, stop=1
-    )
-
-    # test descriptor
-    data_key_names: dict[str, str] = docs["descriptor"][0]["object_keys"]["panda"]
-    assert data_key_names == ["x", "y"]
-    for data_key_name in data_key_names:
-        assert (
-            docs["descriptor"][0]["data_keys"][data_key_name]["source"]
-            == "mock+soft://panda-data-hdf_directory"
-        )
-
-    # test stream resources
-    for dataset_name, stream_resource, data_key_name in zip(
-        ("x", "y"), docs["stream_resource"], data_key_names, strict=False
-    ):
-        assert stream_resource == {
-            "run_start": docs["start"][0]["uid"],
-            "uid": ANY,
-            "data_key": data_key_name,
-            "mimetype": "application/x-hdf5",
-            "uri": "file://localhost" + str(tmp_path / "test-panda.h5"),
-            "parameters": {
-                "dataset": f"/{dataset_name}",
-                "swmr": False,
-                "multiplier": 1,
-                "chunk_shape": (1024,),
-            },
-        }
-        assert "test-panda.h5" in stream_resource["uri"]
-
-    # test stream datum
-    for stream_datum in docs["stream_datum"]:
-        assert stream_datum["descriptor"] == docs["descriptor"][0]["uid"]
-        assert stream_datum["seq_nums"] == {
-            "start": 1,
-            "stop": 2,
-        }
-        assert stream_datum["indices"] == {
-            "start": 0,
-            "stop": 1,
-        }
-        assert stream_datum["stream_resource"] in [
-            sd["uid"].split("/")[0] for sd in docs["stream_datum"]
-        ]
-
-
-async def test_hdf_panda_hardware_triggered_flyable_with_iterations(
```

Comment:
I have deleted `test_hdf_panda_hardware_triggered_flyable_with_iterations`  and moved it to
 ```
@pytest.mark.parametrize("number_of_frames", [[1, 1, 1, 1], [2, 3, 100, 3]])
async def test_hardware_triggered_flyable(
```

As we are removing iterations